packages/debugger/webpack/webpack.config-test.js

```diff
@@ -28,7 +28,7 @@ module.exports = merge(commonConfig, {
             plugins: ["@babel/plugin-transform-runtime"]
           }
         },
-        exclude: [path.resolve(__dirname, "..", "node_modules")],
+        exclude: path.resolve(__dirname, "..", "node_modules"),
```

Comment:
Is this equivalent? `exclude` was previously an Array

Reply:
Oh this must have been a mistake on my part. I'll change it back, I assume it is equivalent? I think I've seen both cases.